c++/src/capnp/rpc-twoparty-test.c++

```diff
@@ -789,6 +791,52 @@ KJ_TEST("promise cap resolves between starting request and sending it") {
   KJ_EXPECT(n2 == 1, n2);
 }
 
+KJ_TEST("read error propagates to write error") {
```

Comment:
Is this test name backwards? The error on the write is being propagated to the read.

Obviously not very important though, it's just a test name.

Reply:
You're right. And getting things like this right is important, to avoid confusing people later.